### DISCUSSION TITLE
retro-transposition

### DISCUSSION THREAD

**Discussion by kurt-rhee** [pvlib/pvlib-python#1510] [Q&A] [answered]
Hey all, 

I was wondering if the ability to go retro-transpose POA information exists in pvlib.  For example, if I have POA, surface azimuth and surface tilt could I back out GHI and DHI?

If this doesn't exist yet, is it something that would make a good addition to the library?

**Comment by cwhanse** [comment, marked as answer]
It is available in `pvlib.irradiance.gti_dirint`. There's no unique inverse function POA -> GHI; one has to assume a model for sky diffuse irradiance component, hence the name `gti_dirint` which uses the DIRINT model for sky diffuse.

**Reply by kurt-rhee** [reply]
Awesome, thank you @cwhanse!
